### PR TITLE
Always log outbound responses if inbound enabled

### PR DESCRIPTION
**Pull request by stavros-wb**

### CODE REVIEW

**Review by SokratisVidros — APPROVED** [review]
Super!

**Review by anastasios-workable — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -39,10 +39,10 @@ module.exports = {
         if (typeof errors.callback === 'function') {
           errors.callback(ctx, err);
         }
-      }
-
-      if (inbound.enabled) {
-        utils.safeExec(() => loggable.emit(EVENT.OUTBOUND_RESPONSE, { ctx }), logger);
+      } finally {
```
Comment (nikostoulas) [inline index.js:42]: @stavros-wb What was the case that the outbound response was not logged?

**Comment by alkiskal** [comment]
Are we sure we want this? I mean shouldn't these two be controlled by separate variables? I would guess that someone would want to be able to log only the inbound or outbound. It seems a little bit confusing like this.

**Review by nikostoulas — APPROVED** [review]

**Comment by stavros-wb** [comment]
@alkiskal 
> Are we sure we want this? I mean shouldn't these two be controlled by separate variables? I would guess that someone would want to be able to log only the inbound or outbound. It seems a little bit confusing like this.

the logic remains as was `if(inbound.enabled)` log both start of request and end of request. indeed the commit message could be better :P

@nikostoulas 
> What was the case that the outbound response was not logged?

two conditions
```
a. an error was thrown
b. no error callback was declared in the config, so the default (ctx, err) => throw err was applied
```
All this does, is if for whatever reason an unhandle error is thrown, do log the end for the inbound request

**Review by alkiskal — APPROVED** [review]

**Comment by nikostoulas** [comment]
Thanks for clarifying this @stavros-wb!
Super let's deploy this as a patch version.

**Pull request merged**